content/concepts/scaling-jobs/content.md

```diff
@@ -8,14 +8,43 @@ title = "Scaling Jobs"
   sticky = true
 +++
 
-### Overview
-
-TODO
-
-### ScaledObject spec
-
-TODO
-
-### Example
-
-TODO
\ No newline at end of file
+## Overview
+
+As an alternate to [scaling event driven code as deployments](/concepts/scaling-deployments) you can also run and scale your code as Kubernetes Jobs.  The primary reason to consider this option is to handle processing long running executions.  Rather than processing multiple events within a deployment, for each detected event a single Kubernetes Job is scheduled.  That job will initialize, pull a single event from the message source, and process to completion and terminate.  
+
+For example, if you wanted to use KEDA to run a job for each message that lands on a RabbitMQ queue, the flow may be:
+
+1. When no messages are awaiting processing, no jobs are created.
+1. When a message arrives on the queue, KEDA creates a job.
+1. When the job starts running, it pulls *a single* message and processes it to completion.
+1. As additional messages arrive, additional jobs are created.  Each job processes a single message to completion.
+
+## ScaledObject spec
+
+This specification describes the `ScaledObject` custom resource definition which is used to define how KEDA should scale your application and what the triggers are.
+
+[`scaledobject_types.go`](./../pkg/apis/KEDA/v1alpha1/scaledobject_types.go)
+
+```yaml
+apiVersion: keda.k8s.io/v1alpha1
+kind: ScaledObject
+metadata:
+  name: {scaled-object-name}
+spec:
+  scaleType: job
```

Comment:
This is not in the deployment docs, do we have a type for that as well?

 Would be good to add it there for sake of completeness as well

Reply:
The behavior appears to be default deployment, and opt into others.

Reply:
Agreed, but might be good to show you can specify it to make it more explicit?

Anyhow, not required.